helm/prometheus-rules/templates/alerting-rules/up.all.rules.yml

```diff
@@ -39,8 +39,8 @@ spec:
         cancel_if_cluster_status_deleting: "true"
         cancel_if_kubelet_down: "true"
         cancel_if_cluster_has_no_workers: "true"
-        cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
-        severity: notify
+        cancel_if_outside_working_hours: "true"
```

Comment:
I think we keep the default value,

Reply:
You mean, this alert (`CadvisorDown`) should page outside working hours?
Makes sense, I'll update it.